### PR TITLE
Main Menu PR

### PR DESCRIPTION
**Pull request by HusseinAljanaby**
- Placeholder Game scene.
 - Placeholder wireframe Settings scene.
 - Placeholder wireframe Load Game scene.
 - Quit button.

### CODE REVIEW

**Comment by Gonj3** [comment]
looks good

**Review by Gonj3 — APPROVED** [review]

File: Load.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+using Godot;
+using System;
+
+public partial class Load : Control
+{
+	//Main Menu scene source.
+	private String mainScene = "res://MainMenu.tscn";
```
Comment (jamesp6784) [inline Load.cs:7]: Naming: class fields should be PascalCase
also these strings could be const (same in other files)

File: Menu.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+using Godot;
+using System;
+
+public partial class Menu : Control
+{
+	//TODO ADD GAME SCENE SOURCE.
+	private String gameScene = "res://Game.tscn";
```
Comment (jamesp6784) [inline Menu.cs:7]: See other comment

File: Settings.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+using Godot;
+using System;
+
+public partial class Settings : Control
+{
+	//Main Menu scene source.
+	private String mainScene = "res://MainMenu.tscn";
```
Comment (jamesp6784) [inline Settings.cs:7]: See other comment

**Review by jamesp6784 — APPROVED** [review]
Just some convention stuff

**Pull request merged**